Libraries/Opc.Ua.Client/TraceableSession.cs

```diff
@@ -61,7 +60,7 @@ public TraceableSession(ISession session)
         /// Activity Source static instance.
         /// </summary>
         public static ActivitySource ActivitySource => s_activitySource.Value;
-        private static readonly Lazy<ActivitySource> s_activitySource = new Lazy<ActivitySource>(() => new ActivitySource(ActivitySourceName));
+        private static readonly Lazy<ActivitySource> s_activitySource = new Lazy<ActivitySource>(() => new ActivitySource(ActivitySourceName, "1.0.0"));
 
         /// <summary>
         /// The ISession which is being traced.
```

Comment:
traceablesession should hold on the decorated session even when disposed, that behavior causes some null exceptions / back compat issues with existing projects

Reply:
fixed, but requires Nuget update